### ISSUE TITLE
Segmentation fault on Msys2

### ISSUE THREAD

**Issue by NadAlaba** [fonttools/fonttools#3447]
Fresh install of fonttools on MSYS2 windows 22H2 will cause segmentation fault just after `fonttools -h`

**Comment by anthrotype** [comment]
Can you try to install the pure-python wheel instead of the windows binary wheel?

Do `pip uninstall fonttools` and then `pip install --no-binary=fonttools fonttools`

It might be that the cythonized modules (e.g. cu2qu) don't work with MSYS2 python, I honestly don't know.

**Comment by NadAlaba** [comment]
Trying `pip install --no-binary=fonttools fonttools` produces the same segmentation fault, even with `SETUPTOOLS_USE_DISTUTILS=stdlib` (alhtough no building errors were shown).
Trying to install the [built](https://packages.msys2.org/package/mingw-w64-x86_64-python-fonttools) package with pacman produces the same result, too.
There are [known issues](https://www.msys2.org/docs/python/#known-issues) with CPython, but I thought building the wheel would solve the issue. I'm confused...

**Comment by anthrotype** [comment]
hm try to force pip to download the pure python wheel instead of --no-binary which downloads and builds from the source distribution:

`pip install --platform none --no-deps fonttools`

**Comment by NadAlaba** [comment]
> `pip install --platform none --no-deps fonttools`

Unfortunately `pip install --platform none --no-deps fonttools -t ~` and then copying files manually, produces the same result!

**Comment by anthrotype** [comment]
What do you mean "moving the files"? Move where? Did you uninstall/remove the previous copy first?

**Comment by NadAlaba** [comment]
Yeah, I first uninstalled the packages then reinstalled with `pip`. After install I had to copy `bin` directory to `/mingw64/bin` and `fontTools` and `fonttools-4.48.1.dist-info` to `D:\dev\msys64\mingw64\lib\python3.11\site-packages` in order for the package to import correctly.

However, I did a fresh install of MSYS2 and installed the packge with pacman only using `pacman -S mingw-w64-ucrt-x86_64-python-fonttools`, and now it works correctly with no segfaults. Turned out my environment was polluted with the prebuilt python package, and a fresh install was needed.
